### PR TITLE
Elements: Buttons (tweaks)

### PR DESCRIPTION
**Pull request by talbs**

### CODE REVIEW

File: _posts/elements/2015-04-09-buttons.md [hunk 1]
```diff
@@ -17,77 +17,77 @@ description:  A collection of buttons available for using within the edX platfor
 info: Buttons should be used for performing actions within the edX environment. While we supply a button that looks like a link, it should only really be used for very tertiary actions. edX offers three button sizes, each with normal, hover, active/pressed, and disabled states. We also offer a range of other buttons to suit your needs.
 ---
 
+<h3 class="hd-6 example-set-hd">Default</h3>
 <section class="example-set">
-    <h3 class="hd-6">Default</h3>
```
Comment (clrux) [inline _posts/elements/2015-04-09-buttons.md]: Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.

File: _posts/elements/2015-04-09-buttons.md [hunk 2]
```diff
@@ -17,77 +17,77 @@ description:  A collection of buttons available for using within the edX platfor
...
     <button type="button" class="btn btn-default btn-base">Default Base</button>
     <button type="button" class="btn btn-default btn-large">Default Large</button>
     <button type="button" class="btn btn-default btn-small">Default Small</button>
     <button type="button" class="btn btn-default btn-base" disabled>Default Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Default (Alt)</h3>
 <section class="example-set">
-    <h3 class="hd-6">Default (Alt)</h3>
```
Comment (clrux) [inline _posts/elements/2015-04-09-buttons.md]: Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.

File: _posts/elements/2015-04-09-buttons.md [hunk 3]
```diff
@@ -17,77 +17,77 @@ description:  A collection of buttons available for using within the edX platfor
...
     <button type="button" class="btn btn-default-alt btn-base">Default Base</button>
     <button type="button" class="btn btn-default-alt btn-large">Default Large</button>
     <button type="button" class="btn btn-default-alt btn-small">Default Small</button>
     <button type="button" class="btn btn-default-alt btn-base" disabled>Default Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Primary</h3>
 <section class="example-set">
-    <h3 class="hd-6">Primary</h3>
```
Comment (clrux) [inline _posts/elements/2015-04-09-buttons.md]: Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.

File: _posts/elements/2015-04-09-buttons.md [hunk 4]
```diff
@@ -17,77 +17,77 @@ description:  A collection of buttons available for using within the edX platfor
...
     <button type="button" class="btn btn-primary btn-base">Primary Base</button>
     <button type="button" class="btn btn-primary btn-large">Primary Large</button>
     <button type="button" class="btn btn-primary btn-small">Primary Small</button>
     <button type="button" class="btn btn-primary btn-base" disabled>Primary Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Secondary</h3>
 <section class="example-set">
-    <h3 class="hd-6">Secondary</h3>
```
Comment (clrux) [inline _posts/elements/2015-04-09-buttons.md]: Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.

File: _posts/elements/2015-04-09-buttons.md [hunk 5]
```diff
@@ -17,77 +17,77 @@ description:  A collection of buttons available for using within the edX platfor
...
     <button type="button" class="btn btn-secondary btn-base">Secondary Base</button>
     <button type="button" class="btn btn-secondary btn-large">Secondary Large</button>
     <button type="button" class="btn btn-secondary btn-small">Secondary Small</button>
     <button type="button" class="btn btn-secondary btn-base" disabled>Secondary Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Secondary (Alt)</h3>
 <section class="example-set">
-    <h3 class="hd-6">Secondary (Alt)</h3>
```
Comment (clrux) [inline _posts/elements/2015-04-09-buttons.md]: Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.

File: _posts/elements/2015-04-09-buttons.md [hunk 6]
```diff
@@ -17,77 +17,77 @@ description:  A collection of buttons available for using within the edX platfor
...
     <button type="button" class="btn btn-secondary-alt btn-base">Secondary Base</button>
     <button type="button" class="btn btn-secondary-alt btn-large">Secondary Large</button>
     <button type="button" class="btn btn-secondary-alt btn-small">Secondary Small</button>
     <button type="button" class="btn btn-secondary-alt btn-base" disabled>Secondary Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Links</h3>
 <section class="example-set">
-    <h3 class="hd-6">Links</h3>
```
Comment (clrux) [inline _posts/elements/2015-04-09-buttons.md]: Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.

File: _posts/elements/2015-04-09-buttons.md [hunk 7]
```diff
@@ -17,77 +17,77 @@ description:  A collection of buttons available for using within the edX platfor
...
     <button type="button" class="btn btn-link btn-base">Link Base</button>
     <button type="button" class="btn btn-link btn-large">Link Large</button>
     <button type="button" class="btn btn-link btn-small">Link Small</button>
     <button type="button" class="btn btn-link btn-base" disabled>Link Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Elevated</h3>
 <section class="example-set">
-    <h3 class="hd-6">Elevated</h3>
```
Comment (clrux) [inline _posts/elements/2015-04-09-buttons.md]: Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.

File: _posts/elements/2015-04-09-buttons.md [hunk 8]
```diff
@@ -17,77 +17,77 @@ description:  A collection of buttons available for using within the edX platfor
...
     <button type="button" class="btn btn-elevated btn-base">Elevated Base</button>
     <button type="button" class="btn btn-elevated btn-large">Elevated Large</button>
     <button type="button" class="btn btn-elevated btn-small">Elevated Small</button>
     <button type="button" class="btn btn-elevated btn-base" disabled>Elevated Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Combo</h3>
 <section class="example-set">
-    <h3 class="hd-6">Combo</h3>
```
Comment (clrux) [inline _posts/elements/2015-04-09-buttons.md]: Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.

File: _posts/elements/2015-04-09-buttons.md [hunk 9]
```diff
@@ -17,77 +17,77 @@ description:  A collection of buttons available for using within the edX platfor
...
-    <h3 class="hd-6">With Icons</h3>
+<!--<h3 class="hd-6 example-set-hd">With Icons</h3>
+<section class="example-set">
     <button type="button" class="btn btn-default btn-base has-icon">Default Base with Icon</button>
     <button type="button" class="btn btn-default btn-base has-icon icon-reversed">Default Base with Icon Reversed</button>
 </section>-->
 
+<h3 class="hd-6 example-set-hd">Overlay</h3>
 <section class="example-set">
-    <h3 class="hd-6">Overlay</h3>
```
Comment (clrux) [inline _posts/elements/2015-04-09-buttons.md]: Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.

File: _posts/elements/2015-04-09-buttons.md [hunk 10]
```diff
@@ -96,8 +96,8 @@ info: Buttons should be used for performing actions within the edX environment.
     </div>
 </section>
 
+<h3 class="hd-6 example-set-hd">Links</h3>
 <section class="example-set">
-    <h3 class="hd-6">Links</h3>
```
Comment (clrux) [inline _posts/elements/2015-04-09-buttons.md]: Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.

File: _src/static/sass/components/_buttons.scss [hunk 11]
```diff
@@ -259,33 +265,13 @@
 // #LINK
 // ----------------------------
 .btn-link {
```
Comment (clrux) [inline _src/static/sass/components/_buttons.scss]: Don't we want the "beefiness" of these so they align with buttons more easily?

File: _src/static/sass/utilities/_variables.scss [hunk 12]
```diff
@@ -322,7 +320,7 @@ $timing: (
     x-slow:     1.5s,
     fast:       0.5s,
     x-fast:     0.25s,
-    xx-fast:    0.1s
+    xx-fast:    0.125s
```
Comment (clrux) [inline _src/static/sass/utilities/_variables.scss]: Slowin' me down, bruh.

File: _src/static/sass/components/_buttons.scss (hunk 11 above)
Comment (talbs) [inline _src/static/sass/components/_buttons.scss]: Hmm, I was hoping that could be done through ``display: inline-block;`` and ``vertical-align: middle;`` when we're doing the more complex patterns, like form actions or other places where we'd use buttons side-by-side. Cool to wait till then?

**Comment by talbs** [comment]
@clrux, I've addressed all of your feedback (thanks for that). Let me know if things are good to go now.

File: _src/static/sass/components/_buttons.scss (hunk 11 above)
Comment (clrux) [inline _src/static/sass/components/_buttons.scss]: Yep, thanks.

**Comment by clrux** [comment]
:+1:

**Pull request merged**